docs_src/design/ucr/0001-System-Events-for-Devices.md

```diff
@@ -0,0 +1,49 @@
+## System Events for Devices
+### Submitters
+- Lenny Goodell (Intel)
+
+### Change Log
+- [pending](https://github.com/edgexfoundry/edgex-docs/pull/779) (2022-06-22)
+
+### Market Segments
+- All or Any segments that can take advantage of auto on-boarding of new devices and auto start some AI/ML processing on the data from the new devices.
```

Comment:
It isn't just about "**auto** on-boarding of **new** devices", but rather about any device on-boarding/decommissioning information (added/updated/deleted).